app/routes/bdb/components/BdbDetail.tsx

```diff
@@ -11,6 +11,7 @@ import {
 import { usePreparedEffect } from '@webkom/react-prepare';
 import { isEmpty } from 'lodash';
 import { Trash2 } from 'lucide-react';
+import moment from 'moment';
```

Comment:
hehe we actually use `moment-timezone`

```suggestion
import moment from 'moment-timezone';
```

Reply:
Fixed this now :+1: